apps/vr-tests-react-components/src/stories/Breadcrumb/Breadcrumb.stories.tsx

```diff
@@ -12,10 +12,7 @@ export default {
 export const Appearance = () => (
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=29554&commitId=d650fbfc03371715e9c631b5eede6c9eaed52a2b&env=prod&clientType=FLUENTUI)


  <details><summary><b>Breadcrumb Converged</b> 24 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Breadcrumb Converged.appearance - High Contrast.pressed.chromium.png|234933| Changed|
|Breadcrumb Converged.appearance - Dark Mode.focused.chromium.png|197276| Changed|
|Breadcrumb Converged.appearance - High Contrast.default.chromium.png|234933| Changed|
|Breadcrumb Converged.appearance - RTL.default.chromium.png|1664| Changed|
|Breadcrumb Converged.appearance - RTL.hover.chromium.png|1664| Changed|
|Breadcrumb Converged.appearance - Dark Mode.pressed.chromium.png|197276| Changed|
|Breadcrumb Converged.appearance - High Contrast.hover.chromium.png|234933| Changed|
|Breadcrumb Converged.appearance - Dark Mode.hover.chromium.png|197276| Changed|
|Breadcrumb Converged.appearance.hover.chromium.png|1676| Changed|
|Breadcrumb Converged.appearance - Dark Mode.default.chromium.png|197276| Changed|
|Breadcrumb Converged.appearance - RTL.focused.chromium.png|1664| Changed|
|Breadcrumb Converged.appearance - High Contrast.focused.chromium.png|234933| Changed|
|Breadcrumb Converged.appearance.focused.chromium.png|1676| Changed|
|Breadcrumb Converged.appearance.default.chromium.png|1676| Changed|
|Breadcrumb Converged.appearance - RTL.pressed.chromium.png|1664| Changed|
|Breadcrumb Converged.appearance.pressed.chromium.png|1676| Changed|
|Breadcrumb Converged.divider type.hover.chromium.png|506| Changed|
|Breadcrumb Converged.divider type.pressed.chromium.png|506| Changed|
|Breadcrumb Converged.divider type.focused.chromium.png|506| Changed|
|Breadcrumb Converged.divider type.default.chromium.png|506| Changed|
|Breadcrumb Converged.size.hover.chromium.png|990| Changed|
|Breadcrumb Converged.size.focused.chromium.png|990| Changed|
|Breadcrumb Converged.size.pressed.chromium.png|990| Changed|
|Breadcrumb Converged.size.default.chromium.png|990| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>